### DISCUSSION TITLE
Any chance on getting this to run on Windows 10?

### DISCUSSION THREAD

**Discussion by jtotri** [mayrthom/CartridgeWriter_uPrint#8] [Q&A] [answered]
Has anyone figured out how to get this to run on Windows 10? Thanks for any insight you can offer :)

**Comment by mayrthom** [comment]
Thank you for opening this discussion.
I was pretty sure that it also works in Windows 10.  So I tried it myself to download and open it on an windows 10 Laptop and it failed.
Then I realised that the error only occurs when downloading just the .exe file from the repository.
So I added a release file, to solve this problem.
If you download the whole repository as zip or you download the release file (v1.4.1) on the right hand side under "Releases" it shoud work perfectly fine.

Let me know if it works.

Best Regards

Thomas

**Comment by jtotri** [comment, marked as answer]
Thank you so much for the assist Thomas! I'm new to this printer hacking
game and just noticed this program is for uPrint printers while I have a
stratasys dimension sst printer. The program works on windows 10 now but I
wasted your time because I have the wrong printer type 😔 My apologies.
